src/bp/index.ts

```diff
@@ -23,6 +22,9 @@ const printPlainError = err => {
 global.DEBUG = Debug
 global.printErrorDefault = printPlainError
 
+// eslint-disable-next-line import/order
+import { runNluServerWithArgv } from './nlu' // needs DEBUG variable
```

Comment:
I would suggest removing the eslint comment and use `const { runNluServer } = require('./nlu')` like we do for other files in this file

Reply:
sure :+1: